products/deploymentmanager/api.yaml

```diff
@@ -0,0 +1,177 @@
+          the deployment will fail. Note that updating this field does not
+          actually affect the deployment, just how it is updated.
+        url_param_only: true
+        default_value: :CREATE_OR_ACQUIRE
+        values:
+          - :ACQUIRE
+          - :CREATE_OR_ACQUIRE
+      - !ruby/object:Api::Type::Enum
+        name: 'deletePolicy'
+        description: |
+          Set the policy to use for deleting new resources on update/delete.
+          Valid values are `DELETE` (default) or `ABANDON`. If `DELETE`,
+          resource is deleted after removal from Deployment Manager. If
+          `ABANDON`, the resource is only removed from Deployment Manager
+          and is not actually deleted. Note that updating this field does not
+          actually change the deployment, just how it is updated.
+        url_param_only: true
+        default_value: :DELETE
+        values:
+          - :ABANDON
+          - :DELETE
+      - !ruby/object:Api::Type::Boolean
+        name: 'preview'
+        description: |
+          If set to true, a deployment is created with "shell" resources
+          that are not actually instantiated. This allows you to preview a
+          deployment. It can be updated to false to actually deploy
+          with real resources.
+        default_value: false
+        url_param_only: true
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'name'
+        description: |
+          Unique name for the deployment
+        required: true
+      - !ruby/object:Api::Type::String
+        name: 'description'
+        description: |
+          Optional user-provided description of deployment.
+      - !ruby/object:Api::Type::Fingerprint
+        name: 'fingerprint'
+        description: |
+          Output only. Fingerprint generated by Deployment Manager for use in
+          modification requests.
+      - !ruby/object:Api::Type::Array
+        name: 'labels'
+        description: |
+          Key-value pairs to apply to this address.
+        send_empty_value: true
+        item_type: !ruby/object:Api::Type::NestedObject
+          properties:
+            - !ruby/object:Api::Type::String
+              name: 'key'
+              description: |
+                Key for label.
+            - !ruby/object:Api::Type::String
+              name: 'value'
+              description: |
+                Value of label.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'target'
+        description: |
+          Parameters that define your deployment, including the deployment
+          configuration and relevant templates.
+        properties:
+          - !ruby/object:Api::Type::NestedObject
+            name: 'config'
+            description: |
+              The root configuration file to use for this deployment.
+            properties:
+              - !ruby/object:Api::Type::String
+                name: 'content'
```

Comment:
i figured it wasn't a big deal since the imports show that, but I can always add more examples. I'd actually rather not make a test out of it, unless you feel strongly that we should test that behavior

Reply:
I'd rather we test it, since I think it's how most users will use the resource.

Reply:
👍